tests/home_work.spec.js

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from '@playwright/test';
+   
+     test('Perform Login', async ({ page }) => {
+      await page.goto('https://www.saucedemo.com/');
+      await page.locator('[data-test="username"]').click();
+      await page.locator('[data-test="username"]').fill('standard_user');
```

Comment:
Please, add   testIdAttribute to the playwright config file like : 
```
use: {
    testIdAttribute: 'data-test',
...
}
```
Please, read more: https://playwright.dev/docs/api/class-testoptions#test-options-test-id-attribute